release/nightly_tests/dataset/pipelined_ingestion_app.yaml

```diff
@@ -0,0 +1,17 @@
+base_image: "anyscale/ray-ml:pinned-nightly-py37-gpu"
+env_vars: {}
+
+python:
+  pip_packages: []
+  conda_packages: []
+
+post_build_cmds:
+  - pip uninstall -y numpy ray || true
+  - sudo rm -rf /home/ray/anaconda3/lib/python3.7/site-packages/numpy
+  - pip install numpy || true
+  - pip3 install -U {{ env["RAY_WHEELS"] | default("ray") }}
+  - pip install -U git+https://github.com/ray-project/ray_shuffling_data_loader.git@add-embedding-model
```

Comment:
We may want to eventually move this embedding model into `release/nightly_tests/dataset`, since this `ray_shuffling_data_loader` dependency really shouldn't be necessary.

Reply:
![image](https://user-images.githubusercontent.com/837180/134614205-4a2c822c-f9d1-481b-89ba-be2634c8c49b.png)